cryptol-saw-core/src/Verifier/SAW/Cryptol.hs

```diff
@@ -10,11 +10,14 @@
 
 {- |
 Module      : Verifier.SAW.Cryptol
-Copyright   : Galois, Inc. 2012-2015
+Copyright   : Galois, Inc. 2012-2025
 License     : BSD3
 Maintainer  : huffman@galois.com
 Stability   : experimental
 Portability : non-portable (language extensions)
+
+This module 'imports' various Cryptol elements (Name,Expr,...),
```

Comment:
Haddock quibble: surrounding a word in single quotes will cause Haddock to try to render it as though it is the name of a Haskell function, but I don't think that was your intent here:

```suggestion
This module \'imports\' various Cryptol elements (Name,Expr,...),
```